app/server/models/transfer_account.py

```diff
@@ -194,19 +194,38 @@ def get_approval(self, receiving_address):
                 return approval
         return None
 
-    def approve_and_disburse(self):
-        if not self.is_approved:
-            self.is_approved = True
+    def approve_and_disburse(self, initial_disbursement=None, auto_resolve=False):
+        from server.utils.access_control import AccessControl
 
-            if self.is_beneficiary:
-                disbursement = self.make_initial_disbursement()
+        auto_approve = current_app.config['AUTO_APPROVE_TRANSFER_ACCOUNTS']
```

Comment:
isn't this now deprecated in favour of the tiering system below? though i guess doesn't hurt to leave in, and just have it set to true

Reply:
maybe we should have `AUTO_APPROVE_TRANSFER_ACCOUNTS` apply to both enrollers and admins, none at all, or everyone. What do you think?

Reply:
I can't see enrollers wanting to be auto approved though...

Reply:
I think it's probably simplest to go for none at all. The logic you explained in the original documentation is very easy to reason with:
- admins and superadmins can always add users without them being approved by a third party, subadmins never can
- only superadmins can disburse non-default amounts without it being approved.

Reply:
ok, will remove then

Reply:
done